lib/beaker-pe/install/pe_utils.rb

```diff
@@ -702,9 +704,20 @@ def setup_defaults_and_config_helper_on(host, master, acceptable_exit_codes=nil)
         #Install PE based on global hosts with global options
         #@see #install_pe_on
         def install_pe
+          options.merge!(pe_presets)
```

Comment:
this block should be at the top of `install_pe_on` rather than `install_pe`, otherwise people using the former won't get these changes. `install_pe` is really just a shortcut, it shouldn't have functionality in it.